subsys/net/lib/lwm2m_client_utils/lwm2m/ecid_signal_meas_info.c

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#define LOG_MODULE_NAME net_lwm2m_obj_signal_meas_info
+#define LOG_LEVEL CONFIG_LWM2M_LOG_LEVEL
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+#include <string.h>
+#include <init.h>
+
+#include "lwm2m_object.h"
+#include "lwm2m_engine.h"
+#include "net/lwm2m_client_utils.h"
```

Comment:
~~This is actually a public header, so we should include it with square brackets <>.~~

Nevermind, I see it's done in the next commit.